### DISCUSSION TITLE
Selenium-Profiles & -Driverless -> Where is the tmp profile saved?

### DISCUSSION THREAD

**Discussion by nicostrebel** [kaliiiiiiiiii/Selenium-Profiles#85] [Q&A] [answered]
I would like to copy the temporary profile for future uses.

Example: 
When i do manage several different accounts i want to simply load the chrome profile in which i have previously logged in with the account to preserve sessions.

My questions:
Where is the temp. profile stored?
How do i load this profile with the driver after i have copied it to an permanent storage?

@kaliiiiiiiiii

**Comment by ttlns** [comment, marked as answer]
> Where is the temp. profile stored? 

you can set the path with `options.add_argument("--user-data-dir=" + user_data_dir)`
and then copy the profile to another directory before `driver.quit()`.
> How do i load this profile with the driver after i have copied it to an permanent storage?

just load it again with `options.add_argument("--user-data-dir=" + user_data_dir)`

@nicostrebel please mark this as the answer, if it resolves your question

**Comment by ttlns** [comment]
Closing as it seems to be resolved
